### PR TITLE
RANCHER-1941

### PR DESCRIPTION
**Pull request by sergii-msn**
RANCHER-1941: Create and set 2 env varfor (e)testing and (e)dev for linked-data module

### CODE REVIEW

**Comment by sergii-msn** [comment]
https://folio-org.atlassian.net/browse/RANCHER-1941

File: resources/helm/development.yaml [hunk 1]
```diff
@@ -1801,7 +1801,11 @@ mod-licenses:
     enabled: true
     maxUnavailable: 1
 mod-linked-data:
-  extraEnvVars: []
+  extraEnvVars:
+    - name: CACHE_TTL_SPEC_RULES_MS
+      value: 0
```
Comment (eldiiar-duishenaliev) [inline resources/helm/development.yaml:1806]: Should it be in quotes?

File: resources/helm/development.yaml [hunk 2]
```diff
@@ -1801,7 +1801,11 @@ mod-licenses:
...
+    - name: CACHE_TTL_SETTINGS_ENTRIES_MS
+      value: 0
```
Comment (eldiiar-duishenaliev) [inline resources/helm/development.yaml:1808]: value: "0" ?

Comment (sergii-msn) [reply resources/helm/development.yaml]: you are right, fixed

File: resources/helm/development.yaml (hunk 1 above)
Comment (sergii-msn) [reply resources/helm/development.yaml]: fixed

**Review by OHaimanov — APPROVED** [review]

**Review by yaroslavishchenko — APPROVED** [review]

**Pull request merged**